Upstream-Status: Pending

RPM no longer includes an internal version of db, it now relies on the system
version.  The system version in Poky is available at /usr/include/db51

Signed-off-by: Mark Hatle <mark.hatle@windriver.com>

rebased to sat-solver commit 9e1f2a097965debebc69cebf8ed73c6ff31a5220

Signed-off-by: Qing He <qing.he@intel.com>

--- git/ext/repo_rpmdb.c.orig	2011-01-06 10:46:37.369900542 -0600
+++ git/ext/repo_rpmdb.c	2011-01-06 10:48:35.482919434 -0600
@@ -29,11 +29,7 @@
 #include <rpm/rpmdb.h>
 
 #ifndef DB_CREATE
-# ifdef FEDORA
-#  include <db.h>
-# else
-#  include <rpm/db.h>
-# endif
+#  include <db.h>
 #endif
 
 #include "pool.h"
